### PR TITLE
Added en_PH unit test folder + AddressTest.php

### PR DESCRIPTION
**Pull request by Newman101**
Added unit test folder and AddressTest.php for ```en_PH``` Provider. 

Thank you.

### CODE REVIEW

File: test/Faker/Provider/en_PH/AddressTest.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Faker\Test\Provider\en_PH;
+
+use Faker\Generator;
+use Faker\Provider\en_PH\Address;
+use Faker\Provider\en_PH\Person;
```
Comment (smithandre) [inline test/Faker/Provider/en_PH/AddressTest.php]: If Im not mistaking, this class is never used and can be removed?

Comment (Newman101) [inline test/Faker/Provider/en_PH/AddressTest.php]: Interesting, I thought I removed it. O.o 

But apparently, I did not.

Sorry for inconvenience.

**Comment by Newman101** [comment]
That's it, the unused class is now removed.

**Comment by Newman101** [comment]
That's a random build fail again: 

1) Faker\Test\Provider\InternetTest::testIpv4NotLocalNetwork
Failed asserting that '1.46.124.204' does not match PCRE pattern "/\A1\./".

**Comment by fzaninotto** [comment]
Thanks

**Pull request merged**